app/models/api/RegistrationInformation.scala

```diff
@@ -35,7 +35,7 @@ object RegistrationInformation {
     (__ \ "regStartDate").read[LocalDate] and
     (__ \ "channel").read[RegistrationChannel] and
       // Default to regStartDate for records created before this field was added
-    (__ \ "lastModified").read[LocalDate].orElse((__ \ "regStartDate").read[LocalDate])
+    (__ \ "lastModified").read[LocalDateTime].orElse((__ \ "regStartDate").read[LocalDateTime])
```

Comment:
You can remove regStartDate and the comment

Also, we will need a runonce method to go through all the reginfo entries and change their date to localdatetime if it's still using the old format. We had an example on the vat reg backend a few versions ago doing a similar thing.